README.md

```diff
@@ -14,4 +14,4 @@ create database objects;
 
 \c objects
 
-create table objects (id integer, online bool, lastseen character varying);
+create table objects (id integer, online bool, lastseen timestamp with time zone);
```

Comment:
What's about migrations? :)

Reply:
Was thinking of focusing on goroutines synchronisation for now, and check out new packages later